integration-tests-aws2/README.adoc

```diff
@@ -9,13 +17,18 @@ To run the tests against the real AWS API, you need to
   (access key ID and secret access key)
 * Export our preferred
   https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions[AWS region].
+* For SES, you additionally need a https://app.mailslurp.com/[MailSlurp account] and API key
+  (the free dev account should be anough)
```

Comment:
Small typo

Reply:
That's the dialect spoken by Slovaks living in South Moravia ;)